diff --git a/node_modules/@electron/get/dist/cjs/index.js b/node_modules/@electron/get/dist/cjs/index.js
index f4d7b23..ef98df6 100644
--- a/node_modules/@electron/get/dist/cjs/index.js
+++ b/node_modules/@electron/get/dist/cjs/index.js
@@ -90,6 +90,7 @@ function downloadArtifact(_artifactDetails) {
                     utils_1.ensureIsTruthyString(artifactDetails, 'version');
                     artifactDetails.version = utils_1.normalizeVersion(artifactDetails.version);
                     fileName = artifact_utils_1.getArtifactFileName(artifactDetails);
+                    console.log('=====url========', url);
                     url = artifact_utils_1.getArtifactRemoteURL(artifactDetails);
                     cache = new Cache_1.Cache(artifactDetails.cacheRoot);
                     if (!!artifactDetails.force) return [3 /*break*/, 2];
