diff --git a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
index 339d76a..e0407e3 100755
Binary files a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework and b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework differ
diff --git a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/Info.plist b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/Info.plist
index 937179d..482a9cb 100644
--- a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/Info.plist	
+++ b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/Info.plist	
@@ -13,7 +13,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleVersion</key>
-	<string>10.1.0</string>
+	<string>10.1.4</string>
 	<key>DTCompiler</key>
 	<string>com.apple.compilers.llvm.clang.1_0</string>
 	<key>DTSDKBuild</key>
diff --git a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/resources.pak b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/resources.pak
index 04844e6..2b9f0dd 100644
Binary files a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/resources.pak and b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/resources.pak differ
diff --git a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/v8_context_snapshot.bin b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/v8_context_snapshot.bin
index 3295b2d..3e28110 100644
Binary files a/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/v8_context_snapshot.bin and b/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources/v8_context_snapshot.bin differ
diff --git a/node_modules/electron/dist/Electron.app/Contents/Info.plist b/node_modules/electron/dist/Electron.app/Contents/Info.plist
index ed369b9..310a8bb 100644
--- a/node_modules/electron/dist/Electron.app/Contents/Info.plist
+++ b/node_modules/electron/dist/Electron.app/Contents/Info.plist
@@ -19,9 +19,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>10.1.0</string>
+	<string>10.1.4</string>
 	<key>CFBundleVersion</key>
-	<string>10.1.0</string>
+	<string>10.1.4</string>
 	<key>DTCompiler</key>
 	<string>com.apple.compilers.llvm.clang.1_0</string>
 	<key>DTSDKBuild</key>
diff --git a/node_modules/electron/dist/version b/node_modules/electron/dist/version
index 3b9bddf..71e35e8 100644
--- a/node_modules/electron/dist/version
+++ b/node_modules/electron/dist/version
@@ -1 +1 @@
-10.1.0
\ No newline at end of file
+10.1.4
\ No newline at end of file
